installer/README.md

```diff
@@ -1,78 +1,59 @@
-# Cloudflow installer
+# CloudFlow Installer
+This project is the CloudFlow Installer, implemented as a Kubernetes operator.
 
-This is the installer for the [Cloudflow](https://github.com/lightbend/cloudflow) toolkit.
+## Development
 
-This installer deploys all the backend components required to turn your Kubernetes cluster into a Cloudflow-compliant platform.
+Required tools:
 
-The Cloudflow installer deploys:
-- The Cloudflow operator, which orchestrates the deployment of Cloudflow applications
-- The [Spark Operator](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator)
-- The [Strimzi Kafka Operator](https://strimzi.io/) used to manage Apache Kafka clusters co-located or pre-existing
-- The [Lyft Flink Operator](https://github.com/lyft/flinkk8soperator)
-- The required service accounts with the minimal permissions needed by the supporting components
+* `sbt`
+* `make`
+* `helm`
+* `kubectl`
+* `gcloud` - with docker [configured to access GCR](https://cloud.google.com/container-registry/docs/advanced-authentication).
 
-Additionally, this installer deploys:
-- NFS - a supporting component that provides a shareable file system to enable storage for stateful applications
+## Installing
 
-Currently, the provided installation scripts are validated for Google Kubernetes Engine (GKE) on the Google Cloud Platform.
-Testing on other major cloud providers is in the roadmap.
+Currently the CloudFlow Installer operator is installed using `kubectl` and a yaml file located in the `/test` directory. This has been tested to work on the following Kubernetes distributions:
```

Comment:
We'll have to update this doc for the public. The Readme should not start with the Development notes.